### PR TITLE
Update news section

### LINKED ISSUE #1321: Update news page to use newsroom API
@shabnammayel The News page currently shows news items that have been manually added to this website using markdown. It would be best to use the newsroom api here but we are going to lose the sidebar elements as those are meant to work with markdowns rather then API data.

Here is what it would look like without the sidebar:
https://deploy-preview-1320--jakartaee.netlify.app/news/

I agree that it looks better with the sidebar but for now we do not support sidear elements with news items coming from newsroom. We would need to write code to implement that.

Let me know if its ok if we lose the sidebar for now and implement the sidebar on a later date.

Thanks.

### PR DESCRIPTION
**Pull request by ericpoirier**
Fixes #1321 

Signed-off-by: Eric Poirier <eric.poirier@eclipse-foundation.org>

### CODE REVIEW

**Comment by linkfang** [comment]
I noticed some of the blogs were showing a different author than those on prod. It seems most blogs from Tanja become Jacob on preview site. Not sure if this is expected?
 
Such as, "Even closer to Jakarta EE 10!" (March 21), it's Tanja on prod and it's from her blog, but on preview, the author is Jacob.

**Comment by chrisguindon** [comment]
> I noticed some of the blogs were showing a different author than those on prod. It seems most blogs from Tanja become Jacob on preview site. Not sure if this is expected?
> 
> Such as, "Even closer to Jakarta EE 10!" (March 21), it's Tanja on prod and it's from her blog, but on preview, the author is Jacob.

I think it is. The author shown here is the person that submitted the news article to the newsroom. However, we are planning to allow newsroom editors to alter the author.

File: js/main.js [hunk 1]
```diff
@@ -98,5 +98,9 @@ document.addEventListener("DOMContentLoaded", function(event) {
         eclipseFdnVideos.replace();
       });
 
+      $(document).on("click", ".pagination li a", function(){
```
Comment (chrisguindon) [inline js/main.js]: I feel this is something that should be fix in solstice assets. I think this is a problem that all site has.

Comment (ericpoirier) [reply js/main.js]: I created the following MR to add the functionality to solstice assets
https://gitlab.eclipse.org/eclipsefdn/it/webdev/solstice-assets/-/merge_requests/316

Comment (chrisguindon) [reply js/main.js]: What is the status of this PR?

Comment (chrisguindon) [reply js/main.js]: Did we address all the feedback from the issue?

Comment (ericpoirier) [reply js/main.js]: Yes, the updates were made in this PR.

**Comment by ericpoirier** [comment]
We have the +1 from Shabnam.

**Review by chrisguindon — APPROVED** [review]

**Pull request merged**